benchmarks/package.json

```diff
@@ -7,17 +7,13 @@
     "bench": "tsx src/index.ts",
     "bench:many": "tsx src/many/index.js",
     "bench:json": "tsx src/json/index.js",
-    "bench:tuple": "tsx src/tuple/index.js",
-    "start": "npm run build && npm run bench"
+    "bench:tuple": "tsx src/tuple/index.js"
   },
   "dependencies": {
     "benny": "^3.7.1",
     "parjs": "0.12.7"
   },
   "devDependencies": {
-    "@types/benchmark": "^2.1.1",
```

Comment:
This is needed for benchmark helpers, because `benny` depends on some `@types/benchmark` types.